lib/active_record/sql_analyzer/monkeypatches/query.rb

```diff
@@ -9,7 +10,7 @@ def execute(sql, *args)
           safe_sql = nil
 
           SqlAnalyzer.config[:analyzers].each do |analyzer|
-            if SqlAnalyzer.config[:should_log_sample_proc].call(analyzer[:name])
+            if _square_private_should_sample_query(analyzer[:name])
```

Comment:
Given this is a public gem, it would be better to not namespace things to Square